uptane-standard.md

```diff
@@ -634,6 +633,8 @@ Each ECU in a vehicle receiving over-the-air updates is either a primary or a se
 
 All ECUs MUST verify image metadata as specified in {{metadata_verification}} before installing an image or making it available to other ECUs. A primary ECU MUST perform full verification ({{full_verification}}). A secondary ECU SHOULD perform full verification if possible. See [Uptane Deployment Considerations](#DEPLOY) for a discussion of how to choose between partial and full verification.
 
+ECUs MUST have a secure source of time. The Uptane deployment considerations ({{DEPLOY}}) describe one way to implement an external time server to cryptographically attest time, but any other source of time that the OEM or Uptane implementor considers to be secure MAY be used. When "loading time" is referenced in procedures in this standard, it should be understood to mean loading into memory the current time (if the ECU has its own secure clock), or the most recent attested time.
```

Comment:
I don't like the phrase "considers to be secure", is there anything more specific we can cite?

Reply:
Maybe [RFC 7384](https://tools.ietf.org/html/rfc7384)? I don't want to do that normatively, though. I think that when it comes down to it, OEMs are going to make choices about time that are not perfectly secure, but are "good enough" by their own internal valuation of the risk. So the best realistic thing we can do here is give some best practices in the deployment considerations.